packages/openapi-generator/src/file-serializer/operation.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { OpenApiOperation } from '../openapi-types';
+import { serializeOperation } from './operation';
+
+describe('serializeOperation', () => {
+  it('serializes operation with path and query parameters', () => {
+    const operation: OpenApiOperation = {
+      operationId: 'getFn',
+      method: 'get',
+      tags: [],
+      pathParameters: [
+        {
+          in: 'path',
+          name: 'id',
+          schema: { type: 'string' },
+          required: true
+        },
+        {
+          in: 'path',
+          name: 'subId',
+          schema: { type: 'string' },
+          required: true
+        }
+      ],
+      queryParameters: [
+        {
+          in: 'query',
+          name: 'limit',
+          schema: { type: 'number' }
+        }
+      ],
+      response: { type: 'string' },
+      // eslint-disable-next-line no-template-curly-in-string
+      pathTemplate: 'test/${id}/${subId}'
+    };
+    expect(serializeOperation(operation)).toMatchInlineSnapshot(`
+      "getFn: (id: string, subId: string, queryParameters?: {'limit'?: number}) => new OpenApiRequestBuilder<string>(
+        'get',
+        \`test/\${id}/\${subId}\`,
```

Comment:
I was also wondering about this point. My opinion on this is that if you have a simple asertion like for the imports it is better to have it visible in the code, because I do not have to go to the snapshop file. So two  or three lines of content. Here it is long engough to use the mathc snapshot I would say.